src/SampleApp/Program.cs

```diff
@@ -117,17 +205,24 @@ static int Main(string[] args)
             _authKey = args[1];
             _secretKey = args[2];
             _advertisingToken = args[3];
+
+            //Note: If using the Integration environment (see https://unifiedid.com/docs/getting-started/gs-environments ), tokens can be generated using: https://example-srvonly-integ.uidapi.com/
+
             if (args.Length >= 5)
             {
                 _domain = args[4];
             }
 
-            ExampleBasicRefresh();
-            ExampleAutoRefresh();
-            ExampleSharing();
+            ExampleBidStreamClient();
```

Comment:
should ExampleAutoRefreshBidStreamClient be put just below ExampleBidStreamClient ?

and add a comment here and say along the line:

_If you are DSP, run ExampleBidStreamClient/ExampleAutoRefreshBidStreamClient()
If you are Sharer, run ExampleSharingCLlient()_

While DSP likely to be SHarer as well, Sharer probably isn't ID_READER and will hit errors